app/uk/gov/hmrc/gform/views/components/HmrcCustomHeader.scala.html

```diff
@@ -0,0 +1,165 @@
+@*
```

Comment:
Why not to use it [`HmrcHeader`](https://github.com/hmrc/play-frontend-hmrc/blob/main/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcHeader.scala.html) ?